github/issues_events.go

```diff
@@ -34,9 +34,13 @@ type IssueEvent struct {
 	//       The Actor committed to master a commit mentioning the issue in its commit message.
 	//       CommitID holds the SHA1 of the commit.
 	//
-	//     reopened, locked, unlocked
+	//     reopened, unlocked
 	//       The Actor did that to the issue.
 	//
+	//     locked
+	//       The Actor locked the issue.
+	//       lock_reason holds reason of locking the issue if provided while locking
```

Comment:
nit: Let's refer to the Go version of the field here (and make it a complete sentence for the godocs):
`LockReason holds the reason of locking the issue (if provided while locking).`